packages/field/src/FormatMixin.js

```diff
@@ -81,13 +80,29 @@ export const FormatMixin = dedupeMixin(
         };
       }
 
-      static get syncObservers() {
-        return {
-          ...super.syncObservers,
-          _onModelValueChanged: ['modelValue'],
-          _onSerializedValueChanged: ['serializedValue'],
-          _onFormattedValueChanged: ['formattedValue'],
-        };
+      _requestUpdate(name, oldValue) {
+        super._requestUpdate(name, oldValue);
+
+        switch (name) {
```

Comment:
how about writing it like that?

```js
const params = 
const map = {
  serializedValue: () => this._onSerializedValueChanged(oldValue),
  formattedValue: () => this._onFormattedValueChanged(oldValue),
};
map[name]();
```

e.g. will also require rewrite for some of the "observer" to accept only the old value (current value should be read by this.foo)

Reply:
Sure :)

Not sure what you mean with the second part? `_onSerializedValueChanged` and `_onFormattedValueChanged` don't do anything with `oldValue` right?

Reply:
sorry forgot the if
```js
if (map[name]) {
  map[name]();
}
```
so it will only call the function if we have something defined for it